cmd/cluster-node-tuning-operator/main.go

```diff
@@ -124,6 +126,10 @@ func operatorRun() {
 		klog.Exit(err)
 	}
 
+	if err := removePerformanceOLMOperator(restConfig); err != nil {
```

Comment:
this make intuitive sense to me, but maybe elaborate a bit more in the commit message about why better to do this in main?

Reply:
Added some more info on the commit message

Reply:
was this resolved?

Reply:
If the current (amended) [commit message](https://github.com/openshift/cluster-node-tuning-operator/pull/346/commits/3130030d8fa1871d419f668b17fce42cac455305) is still not satisfactory let me know.

Reply:
the description of the change is still a bit too terse. Is not obvious what is the benefit of doing this like this, outside of the reconcile loop. Now that everyone has the context is pretty clear, but without some records this knowledge is going to fade away in few months.
The extra description in the commit message doesn't have to be an essay. Few lines describing the why and recording context are sufficient (the "how" is evident from the change itself).

Reply:
ack.  commit message has been modified.